src/main/java/pl/lodz/p/it/zzpj/botsite/entities/UserTask.java

```diff
@@ -0,0 +1,38 @@
+package pl.lodz.p.it.zzpj.botsite.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.DBRef;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.Date;
+
+@Document
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class UserTask {
+    @Id
+    private String id;
+    @DBRef
+    private String botId;
+    @DBRef
+    private String messageId;
+    //TODO UserId can be removed if not necessary
+    @DBRef
+    private String userId;
+    /**
+     * Date of task creation
+     */
+    private Date creationDate;
+    /**
+     * Date of reminder/sending message
+     */
+    private Date reminderDate;
```

Comment:
Zastanawiam się tu, bo jak damy repeatable, to jak ktoś ustawi, że co godzinę, to będzie miało sens jesli trzymamy datę?
Może po prostu trzymać czas (różnicę) od keidy do kiedy ma być, np. data wstępna (LcoalDateTime użyj) + 12 h, więc remindTime wynosiłby jakąś tam odpowiednia wartość, nawet w postaci LocalTime albo LocalDateTime.

Reply:
Ok. Chodziło mi tutaj o to że ktoś np. Ma jakieś zajęcia co tydzień w czwartek ale jeśli ma być możliwość co godzinę to rzeczywiście będzie problem.